tox.ini

```diff
@@ -22,4 +23,6 @@ setenv=
 deps =
     dj-3.2: Django>=3.2,<3.3
     dj-4.1: Django>=4.1,<4.2
```

Comment:
These versions are not officially supported anymore, we could drop them...

Reply:
Thanks will do !